composer.json

```diff
@@ -39,46 +39,28 @@
     "symfony/phpunit-bridge": "^5.4|^6.3"
   },
   "scripts": {
-    "test": [
-      "@lint",
-      "@static-analysis",
-      "@phpunit",
-      "@behat"
-    ],
-    "lint": [
-      "@lint-php",
-      "@lint-composer"
-    ],
-    "lint-php": "vendor/bin/parallel-lint src tests",
-    "lint-composer": "composer validate",
-    "static-analysis": [
-      "@license-headers",
-      "@phpmd",
+    "check": [
```

Comment:
I did not know that before, but `composer check` also runs `composer check-platform-reqs` if it is not defined in composer.json (it matches on the first word then)

Reply:
Not an issue though, just saying

Reply:
For me this is not an issue. `composer check-platform-reqs` still works as ever.

Reply:
it is not an issue, just sharing my (new) knowlegde